core/src/main/java/hudson/Functions.java

```diff
@@ -726,6 +728,20 @@ public static String encode(String s) {
         return Util.encode(s);
     }
 
+    /**
+     * Shortcut function for calling {@link URLEncoder.encode(String,String)} (with UTF-8 encoding).<br>
+     * Useful for encoding URL query parameters in jelly code (as in {@code "...?param=${h.urlEncode(something)}"}).
+     *
+     * @since 2.???
```

Comment:
```suggestion
     * @since TODO
```

Someone has a script that can auto-replace `@since TODO` with the appropriate version number.

Reply:
OK, thanks, good to know. I've pushed-force with this change.